cmd/webhook/main.go

```diff
@@ -246,6 +254,16 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 				},
 			},
 
+			// Messaging
+			messagingv1beta1.Kind("Channel"): {
```

Comment:
I think it should be messagingv1

Reply:
I'm replicating what i see here: https://github.com/knative/eventing/pull/3396/files. Should that PR also be eventingv1 in line 232 or am I misunderstanding something?

Reply:
interesting. I'm looking at this one: https://github.com/knative/eventing/pull/2611/files#diff-58de452513b7e8d8d3cfea23eb4ae6a8R266

and this: https://github.com/knative/eventing/blob/release-0.15/cmd/webhook/main.go#L246

so maybe eventing is wrong?

Reply:
sgtm, changed to `messagingv1` and also changed `eventingv1beta1` -> `eventingv1`